draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -51,15 +51,36 @@ type Props = ButtonProps & {
   iconButton?: boolean
 } & AdditionalContentProps
 
-const GenericButton: React.FunctionComponent<Props> = props => (
-  <span
-    className={classNames(styles.container, {
-      [styles.fullWidth]: props.fullWidth,
-    })}
-  >
-    {props.href && !props.disabled ? renderLink(props) : renderButton(props)}
-  </span>
-)
+export type ButtonFunctions = { focus: () => void }
+
+const getAriaProps = (props: object) => {
```

Comment:
This is copy/pasted in two places. Is there a recommended method for reusing this code? I suppose it's not a big deal, since the "component-library" version is deprecated.

Reply:
It'd be nice to eventually have a package for miscellaneous utils like this. For now, I think this is fine